sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/tasks/TaskManagementSteps.java

```diff
@@ -53,5 +76,152 @@ public TaskManagementSteps(WebDriverHelpers webDriverHelpers) {
+                          .dueDateDateTime(
+                              getLocalDateTimeFromColumns(
+                                  tableRow.get(ColumnHeaders.DUE_DATE.toString())))
+                          .assignedTo(tableRow.get(ColumnHeaders.ASSIGNED_TO.toString()))
+                          .commentsOnExecution(
+                              tableRow.get(ColumnHeaders.COMMENTS_ON_EXECUTION.toString()))
+                          .commentsOnTask(tableRow.get(ColumnHeaders.COMMENTS_ON_TASK.toString()))
+                          .taskStatus(tableRow.get(ColumnHeaders.TASK_STATUS.toString()))
+                          .createdBy(tableRow.get(ColumnHeaders.CREATED_BY.toString()))
+                          .build()));
+        });
+  }
+
+  private List<Map<String, String>> getTableRowsData() {
+    Map<String, Integer> headers = extractColumnHeadersHashMap();
+    List<WebElement> tableRows = getTableRows();
+    List<HashMap<Integer, String>> tableDataList = new ArrayList<>();
+    tableRows.forEach(
+        table -> {
+          HashMap<Integer, String> indexWithData = new HashMap<>();
+          AtomicInteger atomicInt = new AtomicInteger();
+          List<WebElement> tableData = table.findElements(TABLE_DATA);
+          tableData.forEach(
+              dataText -> {
+                webDriverHelpers.scrollToElementUntilIsVisible(dataText);
+                indexWithData.put(atomicInt.getAndIncrement(), dataText.getText());
+              });
+          tableDataList.add(indexWithData);
+        });
+    List<Map<String, String>> tableObjects = new ArrayList<>();
+    tableDataList.forEach(
+        row -> {
+          ConcurrentHashMap<String, String> objects = new ConcurrentHashMap<>();
+          headers.forEach((headerText, index) -> objects.put(headerText, row.get(index)));
+          tableObjects.add(objects);
+        });
+    return tableObjects;
+  }
+
+  private List<WebElement> getTableRows() {
+    webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(COLUMN_HEADERS_TEXT);
+    return baseSteps.getDriver().findElements(TABLE_ROW);
+  }
+
+  private Map<String, Integer> extractColumnHeadersHashMap() {
+    AtomicInteger atomicInt = new AtomicInteger();
+    HashMap<String, Integer> headerHashmap = new HashMap<>();
+    webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(COLUMN_HEADERS_TEXT);
+    webDriverHelpers.waitUntilAListOfWebElementsAreNotEmpty(COLUMN_HEADERS_TEXT);
+    baseSteps
+        .getDriver()
+        .findElements(COLUMN_HEADERS_TEXT)
+        .forEach(
+            webElement -> {
+              webDriverHelpers.scrollToElementUntilIsVisible(webElement);
+              headerHashmap.put(webElement.getText(), atomicInt.getAndIncrement());
+            });
+    return headerHashmap;
+  }
+
+  private LocalDateTime getLocalDateTimeFromColumns(String date) {
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("M/d/yyyy h:mm a");
```

Comment:
"Text '6/7/2021 12:00 AM' could not be parsed at index 15" 
got this during the execution.

Reply:
Strange, I run it now and it is working. 
![Uploading Screenshot 2021-06-07 at 09.40.52.png…]()